routes/route.js

```diff
@@ -0,0 +1,69 @@
+const express = require('express');
+
+const router = express.Router();
+const { pool } = require('../server/db');
+
+router.use(express.json());
+
+router.post('/', async (req, res) => {
+  try {
+    const data = req.body;
+    const newRoute = await pool.query(
+      `INSERT INTO routes (${data.driverId ? 'driver_id, ' : ''} id) 
+      VALUES(${data.driverId ? `${data.driverId}, ` : ''} ${data.id}) RETURNING *;`,
+    );
+    res.status(200).json(newRoute.rows[0]);
+  } catch (err) {
+    console.log(err);
+    res.status(400).send(err.message);
+  }
+});
+
+router.get('/', async (req, res) => {
+  try {
+    const getRoutes = await pool.query(`SELECT * FROM routes;`);
+    res.status(200).json(getRoutes.rows);
+  } catch (err) {
+    console.log(err);
+    res.status(400).send(err.message);
+  }
+});
+
+router.get('/:routeId', async (req, res) => {
+  try {
+    const { routeId } = req.params;
+    const getRoute = await pool.query(`SELECT * FROM routes WHERE id = ${routeId};`);
+    res.status(200).json(getRoute.rows[0]);
```

Comment:
A return body for the response should have `camelCase` keys - use the `keysToCamel` function from `utils` to do this easily for all routes